src/dev/impl/DevToys/ViewModels/Tools/Generators/PasswordGenerator/PasswordGeneratorToolProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Composition;
```

Comment:
Please add `#nullable enable` at the top of the file.

Reply:
done